File: 프로그래머스/unrated/181874. A 강조하기/A 강조하기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string myString) {
    for (int i = 0; i < myString.size(); i++)
    {
        if (myString[i] == 'a') myString[i] = 'A';
        else if (myString[i] >= 'B' && myString[i] <= 'Z') myString[i] += 32;
    }
    return myString;
}

File: 프로그래머스/0/181874. A 강조하기/A 강조하기.java
import java.util.Arrays;
import java.util.Objects;
import java.util.stream.Collectors;

class Solution {
    public String solution(String myString) {
        return Arrays.stream(myString.split(""))
                .map(s -> Objects.equals(s, "a") || Objects.equals(s, "A") ? "A" : s.toLowerCase())
                .collect(Collectors.joining());
    }
}